hack/test-pj.sh

```diff
@@ -1,23 +1,38 @@
 #!/usr/bin/env bash
 # Inspired by https://github.com/kubernetes/test-infra/blob/047b0a0596f7d42b909405c0b8ed4366de2bef72/prow/pj-on-kind.sh
 # Required kubectl
-# Rightnow this work only on the x86 platform(Linux, MacOS)
+# Right now this work only on the x86 platform(Linux, MacOS)
 
 set -o errexit
-set -o nounset
 set -o pipefail
 
 
 function main() {
+  if [[ "$1" == "-h" || "$1" == "--h" || "$1" == "--help" ]];then
+    echo -e "USAGE:\n testpj job branch [purpose]"
```

Comment:
The help message appears as if the branch value is mandatory. I don't see it being used anywhere.

Also the help message can have a simple description about all the parameters displayed.